be/src/runtime/descriptors.cpp

```diff
@@ -108,6 +108,41 @@ std::string TableDescriptor::debug_string() const {
 IcebergTableDescriptor::IcebergTableDescriptor(const TTableDescriptor& tdesc)
         : TableDescriptor(tdesc), _table_location(tdesc.icebergTable.location), _columns(tdesc.icebergTable.columns) {}
 
+HudiTableDescriptor::HudiTableDescriptor(const TTableDescriptor& tdesc, ObjectPool* pool)
```

Comment:
columns/partition_columns/partitions looks like an optional property, I think we'd better check them